src/components/app/App.jsx

```diff
@@ -22,26 +21,24 @@ export default function App() {
           <SearchProvider>
             <PhotoUploadProvider>
               <DetailsProvider>
-                <MyPlantProvider>
-                  <CareLogProvider>
-                    <Header />
-                    <Switch>
-                      <Route exact path="/" component={HomePage} />
-                      <Route
-                        exact
-                        path="/details/:id"
-                        component={DetailsPage}
-                      />
-                      <Route exact path="/my-plants" component={MyPlantsPage} />
-                      <Route exact path="/search" component={SearchPage} />
-                      <Route
-                        exact
-                        path="/my-plants/:common-name"
-                        component={MyCareHistoryPage}
-                      />
-                    </Switch>
-                  </CareLogProvider>
-                </MyPlantProvider>
+                <CareLogProvider>
+                  <Header />
+                  <Switch>
+                    <Route exact path="/" component={HomePage} />
+                    <Route
```

Comment:
In App.js can we decide which format we want to use and do that for all the routes?
I vote for single line.